runner/payload/transferonly/worker.go

```diff
@@ -226,8 +240,25 @@ func (t *transferOnlyPayloadWorker) sendTxs(ctx context.Context) error {
 	txs := make([]*types.Transaction, 0, numAccounts)
 	acctIdx := 0
 
+	randomInt := rand.Uint64()
+
 	for gasUsed < (t.params.GasLimit - 100_000) {
-		transferTx, err := t.createTransferTx(t.privateKeys[acctIdx], t.nextNonce[t.addresses[acctIdx]], t.addresses[(acctIdx+1)%numAccounts], big.NewInt(1))
+
+		dest := t.addresses[(acctIdx+1)%numAccounts]
+		if t.payloadParams.CreateAccounts != nil && *t.payloadParams.CreateAccounts {
+			fakeAddressOffset := t.currFakeAddr
+			t.currFakeAddr++
+
+			var addr common.Address
+			binary.BigEndian.PutUint64(addr[:], fakeAddressOffset)
+
+			binary.BigEndian.PutUint64(addr[8:], randomInt)
+			addr[19] = 0xff
+			addr[18] = 0xff
+			dest = addr
+		}
+
+		transferTx, err := t.createTransferTx(t.privateKeys[acctIdx], t.nextNonce[t.addresses[acctIdx]], dest, big.NewInt(1))
```

Comment:
Just to make sure I understand this change, the transfer payload for benchmarks now creates a new address for each transfer to maximize the modifications to the state trie? So it should now be similar to `account-create`?